<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LazyFormulaClass</class>
 <widget class="QMainWindow" name="LazyFormulaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LazyFormula</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnReload">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewFormula">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete this</string>
    </property>
   </widget>
   <widget class="QLabel" name="labIndexPath">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Index is saved in:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>9</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableFormula">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>471</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <column>
     <property name="text">
      <string>Material</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dosage</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Company</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Percent</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>9</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowInExplore">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>9</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>inExplore</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNewFormula">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>182</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labM">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>All Material:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>numFormula:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labD">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>:All Dosage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>39</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearSearch">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>38</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>labD</zorder>
   <zorder>btnReload</zorder>
   <zorder>btnNewFormula</zorder>
   <zorder>btnDelete</zorder>
   <zorder>labIndexPath</zorder>
   <zorder>listIndex</zorder>
   <zorder>btnSettings</zorder>
   <zorder>tableFormula</zorder>
   <zorder>btnSave</zorder>
   <zorder>btnShowInExplore</zorder>
   <zorder>txtNewFormula</zorder>
   <zorder>labM</zorder>
   <zorder>labF</zorder>
   <zorder>txtSearch</zorder>
   <zorder>btnClearSearch</zorder>
  </widget>
  <widget class="QStatusBar" name="stBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LazyFormula.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClearSearch</sender>
   <signal>clicked()</signal>
   <receiver>txtSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
